cmake/installdata/common/common.txt

```diff
@@ -7,6 +7,7 @@ system/settings/*
 userdata/*
 system/addon-manifest.xml
 system/colors.xml
+system/dialogcolors.xml
```

Comment:
you cannot use the same xml, the colors.xml is used by skins xml and require to have the color name code to be identified, they are limited in his context. The color dialog is different thing and each color have appropriate name as hint to the user

Reply:
or they should be merged, but with a different xml hierarchy to distinguish the two lists

Reply:
Hum, don't mean to disagree but the structure of both files seem to be identical. If the only reason for having a new file is the fact the name  in the new color picker dialog is presented on top of each color I don't consider it a strict requirement. Personally, as a user, I want to pick up a color in the dialog by its visual appearance, i.e. I don't care if the color is called "cerulean". I think those names on the color buttons are not necessary just add complexity to both the user experience or skin porting itself (don't forget whenever a new window is added skins have to implement it and we have a breaking change in the skinning API). That means that we should make an effort to make Skinner's life the easier as possible

Reply:
the name of the colours in the file colours.xml are mandatory to allow GUI system to set the right color when the skin components set a color name instead of an hex,
The color names "may" not be necessary in the new color list i have created, but, this is the only way to give some help to all visually impaired users, having an extra Label in the window dialog xml control is certainly not a huge job for skinners it is only few lines in the xml dialog code.
Then using the existing color list is not appropriate due to CSS color code names that are not possible to change.
so solutions could be to keep two separate files, or else use a single xml file, but keep two different colour lists in it

Reply:
for the information i am visually impaired, and without colour names it would be a problem

Reply:
Some usability/design suggestions:

Maybe color name can be shown next to color rectangle in settings (to know exactly what the current color is without having to enter to change it):

![Captura de pantalla 2021-09-25 193713](https://user-images.githubusercontent.com/58434170/134781259-6033a14a-0b71-4573-b680-c9356a0ed25f.png)

Color name currently highlighted can shown on a label below instead of inside each color:

![Captura de pantalla 2021-09-25 194326](https://user-images.githubusercontent.com/58434170/134781346-f38969c9-e0d8-4bf4-bf84-01ca4018cb7d.png)

Reply:
@CastagnaIT allow me some time to test and play a bit with the code to understand the role of the current colors.xml. The reason I am being "stubborn" with this point is because it always has been a common desire to let scripts provide their own fonts and colours (one of our add-on system biggest limitations). Prior effort has been made (https://github.com/xbmc/xbmc/pull/11136) to move the resolution of colours (and fonts) to the skin object. Subtitle colours are not exactly the same thing (I don't think they should belong to the skin or even the GUI). So let's think and agree on a way forward that allow us to change the design incrementally

Reply:
@enen92 ok for now i do not touch the xml things until you get more info about this or/and also others provide his opinions
@thexai to move the name below it is ok, instead to add the name in the settings list, I wouldn't know how to deal with it, because we cannot store the color name on same setting, this means that every time we should load the xml colors to be allow to find the name for each color setting, would slow down the loading of the settings window

Reply:
Then maybe shown hex value in settings `#RRGGBB` (or the same way it is saved). Also a additional label with hex value can be shown in color picker.

Reply:
I'd agree with @thexai regarding displaying the label over the colour, Due to the length of some of the colour names this results in the label needing to scroll.

![image](https://user-images.githubusercontent.com/5781142/134803213-b38ed323-662d-4a8f-9505-bde040005ed9.png)

Doing a quick play around in the new DialogSelectColor.xml something like this maybe better:

![image](https://user-images.githubusercontent.com/5781142/134803033-8fb687ab-de6e-4d7c-a3b7-8df9d1afa45a.png)

Reply:
Since there is enough space, the value in hex could also be added:

Colour: colour_name - #RRGGBB

Reply:
If I had to pick one option for the color button layout (settings dialog before selecting the colour) the above @thexai suggestion would be the one (instead of colour name) I'd choose. We likely want the user to add custom colours later (either RGB or html) via a new button in the colour picker dialog (requested several times). Forcing the user to provide a name for the provided colour or simply displaying "custom" is worst than having a common way to describe them all, including custom colours

Reply:
if a new color is manually added we display only the hex value not the color name, the user is responsible for his additions

Reply:
in the hex value we have ARGB so will be shown as
Colour: colour_name - #AARRGGBB
maybe there is a way to trim the first two chars with the xml code? or i keep as is?

Reply:
i have applied the changes based on @thexai  @jjd-uk suggestions and updated screenshots